dask/array/tests/test_testing.py

```diff
@@ -7,7 +7,7 @@
 from dask.array.utils import assert_eq
 
 
-@pytest.mark.skipif(sys.flags.optimize, reason="Assertions disabled.")
+@pytest.mark.skipif(bool(sys.flags.optimize), reason="Assertions disabled.")
```

Comment:
Running locally I get `sys.flags.optimize = 0`. When I specify the `-O`/ `-OO` options when running `python` I get `1` and `2`, respectively. Are those all the values this flag can be?

Reply:
Yes. In the tuture there may be >2 (unlikely, but possible). But 0 will continue meaning 0.